README.md

```diff
@@ -53,7 +56,7 @@ In one sentence, explain why.
 
 ## A1.2
 
-«replace this with your answer»
+the parser would only need O(1) memory because it will always check either one or two values regardless of the number of overall values
```

Comment:
# 0

And where are those values stored?

Remember that the parser works using a pushdown stack and a transition table. The list of tiles will need to be stored on that stack (at least partially) so the answer is O(n).